backend/src/test/circuits/adder-full-clear.js

```diff
@@ -0,0 +1,3 @@
+adder = {"gates":[],"circuit":[{"type":"x","time":0,"targets":[3],"controls":[0,1]},{"type":"x","time":1,"targets":[1],"controls":[0]},{"type":"x","time":2,"targets":[3],"controls":[1,2]},{"type":"x","time":3,"targets":[2],"controls":[1]},{"type":"x","time":4,"targets":[1],"controls":[0]}],"qubits":4,"input":[0,0,0,0],"version":1}
```

Comment:
Can you just make this file a JSON instead of js. You can then just import it in as an object by using
``` js
var  adderFullClearCircuit = require("./adder-full-clear.js")
```